tests/integration/spe1_st/test_load_data.py

```diff
@@ -2,12 +2,14 @@
 
 
 @pytest.mark.spe1
+@pytest.mark.xfail(reason="Failing due to enkfs refactor")
```

Comment:
Can you rephrase to something like:
`Fails because ert>5 no longer supports ert-storage`?